src/hooks/protections/received_clone_sync.cpp

```diff
@@ -18,11 +18,36 @@ namespace big
 			return eAckCode::ACKCODE_FAIL;
 		}
 
+		// can be used to crash or reverse sync player data
+		if (dst && dst->m_player_info && dst->m_player_info->m_ped && dst->m_player_info->m_ped->m_net_object
+		    && object_id == dst->m_player_info->m_ped->m_net_object->m_object_id) [[unlikely]]
+		{
+			notify::crash_blocked(src, "player sync");
+			return eAckCode::ACKCODE_FAIL;
+		}
+
+		if (auto object = g_pointers->m_gta.m_get_net_object(*g_pointers->m_gta.m_network_object_mgr, object_id, true)) [[likely]]
+		{
+			if (object->m_object_type == (int)eNetObjType::NET_OBJ_TYPE_PLAYER && object->m_owner_id != src->m_player_id) [[unlikely]]
+			{
+				std::string target = "<UNKNOWN>";
+
+				if (auto tgt = g_player_service->get_by_id(object->m_owner_id))
+					target = tgt->get_name();
+
+				LOGF(stream::net_sync, WARNING, "Rejecting clone sync from {}, who is trying to sync to {}'s player ped", src->get_name(), target);
+				return eAckCode::ACKCODE_FAIL;
+			}
+		}
+
 		auto plyr = g_player_service->get_by_id(src->m_player_id);
 
 		if (plyr && plyr->block_clone_sync) [[unlikely]]
 			return eAckCode::ACKCODE_FAIL;
 
+		if (g_local_player && g_local_player->m_net_object && object_id == g_local_player->m_net_object->m_object_id)
+			LOG(INFO) << "local player sync recieved";
```

Comment:
important log?

Reply:
Who cares? It's a log entry, they're only supposed to be for developers.